### ISSUE TITLE
Space after `constructor`

### ISSUE THREAD

**Issue by BraisGabin** [pinterest/ktlint#1207]
## Expected Behavior
```kotlin
class Klass constructor ()
```

should be flagged and fixed to:

```kotlin
class Klass constructor()
```

## Observed Behavior
It doesn't

## Steps to Reproduce
```shell
echo 'class Klass constructor ()'> Klass.kt && ktlint Klass.kt
```

## Your Environment
* Version of ktlint used: 0.42.1 (same for 0.41.0)
